sections/hybrid_control.tex

```diff
@@ -12,8 +12,9 @@ \section{Network Infrastructure}
 
 The hybrid control system will be network based to make use of the robust protocols which already exist for
 communication over a network. The primary components of the network control system are Ubiquiti Litebeam M5 network bridges for
-long range wireless communication, pictured in Figure \ref{fig:ubiquiti-dish}. This avoids the need for Ethernet cabling
-exceeding 2,000 feet, which is inconvenient to set up.
+long range wireless communication, pictured in Figure \ref{fig:ubiquiti-dish}. These allow operators to send commands to the
```

Comment:
I would keep the mention of Ethernet cabling with your additions! Only because the DTEG shows using Ethernet cable but we're vetoing that recommendation since it's awful to roll out